test/cpu_fusion.cpp

```diff
@@ -1108,6 +1109,88 @@ TEST(cpu_fusion, conv_add)
     EXPECT_TRUE(test::all_close(cpu_results.at(0), int_results.at(0)));
 }
 
+shared_ptr<Function> gen_deconv(const bool add_goe)
+{
+    Shape conv_out_shape{100, 64, 1, 1};
+    auto out_delta = std::make_shared<op::Parameter>(element::f32, conv_out_shape);
+
+    Shape filters_shape{64, 512, 4, 4};
+    Shape bias_shape{512};
+    Shape data_batch_shape{100, 512, 4, 4};
+
+    auto data_label = std::make_shared<pattern::op::Label>(element::f32, data_batch_shape);
+    auto filters = std::make_shared<op::Parameter>(element::f32, filters_shape);
+
+    auto conv = std::make_shared<op::ConvolutionBackpropData>(data_label->get_shape(),
+                                                              filters,
+                                                              out_delta,
+                                                              Strides{1, 1},
+                                                              Strides{1, 1},
+                                                              CoordinateDiff{0, 0},
+                                                              CoordinateDiff{0, 0},
+                                                              Strides{1, 1});
+    auto conv_label = std::make_shared<pattern::op::Label>(conv, nullptr, NodeVector{conv});
+
+    auto mean = std::make_shared<op::Parameter>(element::f32, bias_shape);
+    auto var = std::make_shared<op::Parameter>(element::f32, bias_shape);
+    auto gamma = std::make_shared<op::Parameter>(element::f32, bias_shape);
+    auto beta = std::make_shared<op::Parameter>(element::f32, bias_shape);
+    double eps = 0.001;
+
+    auto goe_bn = std::make_shared<op::GetOutputElement>(conv, 0);
+
+    // Adding a goe will stop fusion since the patterns wont expect to see this op
+    auto bn = add_goe
+                  ? std::make_shared<op::BatchNormInference>(goe_bn, gamma, beta, mean, var, eps)
+                  : std::make_shared<op::BatchNormInference>(conv, gamma, beta, mean, var, eps);
+
+    return make_shared<Function>(NodeVector{bn},
+                                 ParameterVector{filters, out_delta, gamma, beta, mean, var});
+}
+
+// Note: enable deconv tests, when the deconv perf is better than convbackpropdata
+TEST(cpu_fusion, DISABLED_fuse_deconv)
+{
```

Comment:
Instead of disabling, you could explicitly set the environment flag to ensure that the code path is tested https://github.com/NervanaSystems/ngraph/blob/master/test/cpu_test.cpp#L132-L137